otx/algorithms/segmentation/configs/ocr_lite_hrnet_18_mod2/pot_optimization_config.json

```diff
@@ -5,90 +5,89 @@
       "params": {
         "preset": "mixed",
         "target_device": "ANY",
-        "use_fast_bias": false,
```

Comment:
That is strange because we were enabling `use_fast_bias` by removing this attr from config. And there is no `use_fast_bias` in  develop https://github.com/openvinotoolkit/training_extensions/blob/develop/otx/algorithms/segmentation/configs/ocr_lite_hrnet_18_mod2/pot_optimization_config.json

This diff looks broken. Maybe I made previous PR (https://github.com/openvinotoolkit/training_extensions/pull/1696) in a wrong branch?

Reply:
Ahh, I didn't rebase it, one moment